docs/source/example_utils.py

```diff
@@ -25,8 +25,10 @@ def plot_image(image: SimpleITK.Image, cmap: str='gray'):
     size_pixels = image.GetSize()
 
     nda = SimpleITK.GetArrayFromImage(image)
-    plt.figure(figsize=(size_pixels[0] / SCREEN_DPI, size_pixels[1] / SCREEN_DPI))
+    plt.figure(figsize=(size_pixels[1] / SCREEN_DPI, size_pixels[0] / SCREEN_DPI))
```

Comment:
I am finding the images in the other examples a little small. This formula seem to let the size of the figure vary based on the size of the simpleitk image. Do we want that?

Should the images always take up about the same size in the document? 

Also the `dpi` argument is not specified.

Reply:
I wrote that code to get the HelloWorld image to display at about the same size as the sitk.Show, at least with Fiji. It may be that the matplotlib plot_directive is scaling the images, which would be "bad".

Reply:
There is also the image directives matplotlib_plot support such as `:width:` and `:height:` which could effect with how the rendered figure is displayed on the sphinx html document.

Reply:
I guess my goal was to avoid the resampling by creating a display size similar to the original image. But on second thought that isn't really necessary. I tried the :width: and :height: options on HelloWorld, but they didn't seem to have an effect, but I'll try again with FastMarching. I could also just pass in another argument to plot_image() to control it from the RST file.

Reply:
The align is not working as expected either. We should figure out how to get those image directives to work.

Reply:
It strange, the :width: has an effect, although it's after the image creation, so the title ends up shrinking, too. And while :align: right has an effect, it gets partially covered by the code. I also tried putting the whole thing in a Sphinx container, but the plot_directive doesn't seem to want to nest.